website/docs/r/kubernetes_cluster_node_pool.html.markdown

```diff
@@ -98,6 +98,8 @@ The following arguments are supported:
 
 * `gpu_instance` - (Optional) Specifies the GPU MIG instance profile for supported GPU VM SKU. The allowed values are `MIG1g`, `MIG2g`, `MIG3g`, `MIG4g` and `MIG7g`. Changing this forces a new resource to be created.
 
+* `gpu_profile` - (Optional) Specifies whether to install the GPU Driver for the nodes. Possible values are `Install` and `None`. Defaults to `None`.
```

Comment:
According to the API spec's description, "Whether to install GPU drivers. When it's not specified, default is Install.". I believe it should not default to `None`?